massa-execution/src/vm.rs

```diff
@@ -295,12 +294,15 @@ impl VM {
         // created_addr_index is not reset here (it is used at the slot scale)
         context.gas_price = data.gas_price;
         context.max_gas = data.max_gas;
-        context.coins_stack = vec![data.coins];
+        context.stack = vec![StackElement {
+            address: data.sender_address,
+            coins: data.coins,
+            owned_addresses: vec![data.sender_address].into_iter().collect(),
+        }]
+        .into();
```

Comment:
avoid this conversion by changing context.stack to be a vec instead of a vecdeque